package.yaml

```diff
@@ -1,10 +1,14 @@
+# SPDX-FileCopyrightText: 2018-2019 Serokell <https://serokell.io>
+#
+# SPDX-License-Identifier: MPL-2.0
+
 name:                crossref-verifier
 version:             0.1.0.0
-github:              "githubuser/crossref-verifier"
```

Comment:
Why not preserve it (and, yes, fix)?

Reply:
Because I couldn't find it here: https://www.haskell.org/cabal/users-guide/developing-packages.html

Was it supposed to be `source-repository`?

Reply:
It's from here: https://github.com/sol/hpack

Reply:
Ah, sorry, it's `package.yaml`, not `.cabal`.

Reply:
Fixed directly in that commit.